src/comments.js

```diff
@@ -823,11 +830,39 @@ function handleVariableDeclaratorComments(
   return false;
 }
 
+function handleTrailingGraphQLComments(precedingNode, comment) {
+  if (
+    precedingNode &&
+    precedingNode.kind &&
+    (precedingNode.kind === "Field" ||
+      precedingNode.kind === "VariableDefinition")
+  ) {
+    addLeadingComment(precedingNode, comment);
+    return true;
+  }
+  return false;
+}
+
+function handleLeadingGraphQLComments(followingNode, comment) {
+  if (
+    followingNode &&
+    followingNode.kind &&
+    (followingNode.kind === "Field" ||
+      followingNode.kind === "OperationDefinition")
+  ) {
+    addLeadingComment(followingNode, comment);
+    return true;
+  }
+  return false;
+}
```

Comment:
if i dont put the blocks, the comments get attached to the wrong nodes and don't get printed.

i.e.,
```graphql
query { # some comment
  someField
}
```
the comment gets attached to the `{` and not the following field. this can be handled as a special case though — ill remove some of these tests